src/BlazorWebFormsComponents/Component.cs

```diff
@@ -0,0 +1,38 @@
+﻿using BlazorComponentUtilities;
+using BlazorWebFormsComponents.Enums;
+using Microsoft.AspNetCore.Components;
+
+namespace BlazorWebFormsComponents
+{
+	public class Component : BaseWebFormsComponent, IStyle
```

Comment:
`Component` is a very generic name.  Let's give it a more descriptive name to indicate what it is delivering.  Perhaps `BaseStyledComponent` to show that this is a Base Component not to be directly created and delivers the style attributes

Reply:
You are right on this, I was thinking about that during the PR, perhaps the name that you come up with is suitable